roles/kuryr_master/README.md

```diff
@@ -0,0 +1,37 @@
+## OpenStack Kuryr
```

Comment:
We should link to upstream documentation about how Kuryr itself should be configured, as a reference.